### PR TITLE
Refactor documentation

### PR DESCRIPTION
**Pull request by tituspijean**
*April 2022 Docathon*

### CODE REVIEW

**Comment by nathanael-h** [comment]
image
:heavy_check_mark:  review "done" :sweat_smile:

**Review by nathanael-h — APPROVED** [review]

**Comment by tituspijean** [comment]
What do you think @ljf, shall we merge this so that it's out of the way and gives us a better view of the state of the documentation? I'll add some dependencies fixes.

**Comment by tituspijean** [comment]
Draft script for mass edition: https://paste.yunohost.org/yafuquwuse.bash 
@ketsapiwiq

**Comment by ketsapiwiq** [comment]
I tried the script, very quickly it begins saying `git mv` is not well used but I can't understand why.
The `ig` program I couldn't install either on Arch. I tried the script in a fresh Docker Ubuntu image, the result is the same.
Here's the total output: https://paste.yunohost.org/romiruneki.pas
The result looks kinda good tho, for the `*5` thing, I suppose it didn't do half the things it was supposed to do.
Here's the commit with the changes: https://github.com/YunoHost/doc/compare/refactor-admin-guide...ketsapiwiq:doc:refactor-admin-guide

File: pages/02.administer/60.extend/10.configuration_management/configuration_management.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Install unpackaged apps
+template: docs
+taxonomy:
+    category: docs
+routes:
+  default: '/install_unpackaged_apps'
```
Comment (alexAubin) [inline pages/02.administer/60.extend/10.configuration_management/configuration_management.md]: both the title and route are a copypasta of `05.install_unpackaged_app`

**Pull request merged**